### PR TITLE
Get tests to pass in Python 3

### PR DESCRIPTION
**Pull request by astrofrog**
A few notes:

* This does not guarantee everything is working in Python 3, only that the tests pass :)

* Some things could be made more elegant by bundling a copy of ``six.py`` which provides a nicer way to have a Python 2 and 3-compatible code base. Other projects such as Astropy do this. If you would be ok with this, I can update this pull request to include this.

* It would be good to set up Travis for continuous integration to make sure that the tests are always run on all supported Python versions - see #19 for more details

### CODE REVIEW

File: dataverse/dataset.py [hunk 1]
```diff
@@ -1,18 +1,23 @@
 import os
 import json
-import StringIO
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import BytesIO as StringIO
```
Comment (astrofrog) [inline dataverse/dataset.py]: This is an example of an import that could be simplified if we used ``six.py``

File: dataverse/dataset.py [hunk 2]
```diff
@@ -35,7 +40,8 @@ def __init__(self, entry=SWORD_BOOTSTRAP, dataverse=None, edit_uri=None,
         self._id = None
 
         # Updates sword entry from keyword arguments
-        for key, value in kwargs.iteritems():
+        for key in kwargs:
+            value = kwargs[key]
```
Comment (astrofrog) [inline dataverse/dataset.py]: Here, ``six.py`` also helps. The issue is that ``iteritems`` is called ``items`` in Python 3. The ``six.py`` file would provide an iterator called ``six.iteritems`` which does the right thing on all Python versions.

**Comment by rliebz** [comment]
I would be fine either way with `six.py`. I definitely see the benefit of it, although I think that the fixes needed here are few and straightforward enough that the package isn't really suffering in terms of readability.

Is there a reason this prefers relative imports over absolute imports?

**Comment by astrofrog** [comment]
@rliebz - I could switch to all absolute but the way they are at the moment does not work on Python 3. In other packages I've worked on we always use relative, but it's your call :) Just let me know what you would prefer!

**Comment by rliebz** [comment]
I tend to prefer absolute just because it makes readability and refactoring a bit more straightforward. If you could go ahead and make that change, I would be happy to merge this in!

**Comment by astrofrog** [comment]
@rliebz - fixed, and tests pass for me in Python 2.6, 2.7 and 3.4

**Pull request merged**